model/crystal.h

```diff
@@ -700,6 +712,15 @@ namespace dxtbx { namespace model {
       if (!uc_a.is_similar_to(uc_b, uc_rel_length_tolerance, uc_abs_angle_tolerance)) {
         return false;
       }
+      if (postrefined_cell_is_set_) {
```

Comment:
do we want to make the `is_similar_to` comparison take `postrefined` as a parameter? It seems a leap to say because this cell is postrefined the other will also be - will raise `DIALS_ASSERT` if false which is hard to trap gracefully.

Reply:
I think you're right. Perhaps the correct behaviour would be to keep the comparison between "postrefined" cells, but do it only if they are both set. If either one is missing, don't fail.